aten/src/ATen/native/group_norm.cpp

```diff
@@ -120,10 +120,13 @@ std::tuple<Tensor, Tensor, Tensor> native_group_norm_backward(
       at::borrow_from_optional_tensor(gamma_opt);
   const Tensor& gamma = *gamma_maybe_owned;
 
-  bool mixed_type = is_mixed_type(X, mean, rstd);
+  bool mixed_type = is_mixed_type(X, mean, rstd) || is_mixed_type(dY, mean, rstd);
```

Comment:
Hmm, how this change is related  to the memory format problem?

Reply:
It is not related to memory format, and just make sure that X and dY are in bfloat16 and parameters are in float type when it hit mixed data type.

Reply:
Can you please move those to a separate PR and I'll approve it right away?

Reply:
Moved the dtype check